src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -423,10 +434,13 @@ def flow_test(
         else:
             from promptflow._utils.multimedia_utils import persist_multimedia_data
 
+            if stream_output and allow_generator_output and self._executor_proxy.chat_output_name:
```

Comment:
maybe need check whether self._executor_proxy is Nonw?

Reply:
1. executor_proxy is expected to be created in `init`
2. below code also depends on `self._executor_proxy`

but it does look dangerous here. I'll think about how to give it a more clear error message if self._executor_proxy is not initialized here

Reply:
use exposed property instead